api/controller/journal.js

```diff
@@ -21,30 +21,30 @@ var gridfsbucket, CHUNKS_COLL, FILES_COLL;
 // Extract from https://gist.github.com/kongchen/941a652882d89bb96f87
 function _toUTF8(str) {
 	var utf8 = [];
-	for (var i=0; i < str.length; i++) {
+	for (var i = 0; i < str.length; i++) {
 		var charcode = str.charCodeAt(i);
 		if (charcode < 0x80) utf8.push(charcode);
 		else if (charcode < 0x800) {
 			utf8.push(0xc0 | (charcode >> 6),
-			0x80 | (charcode & 0x3f));
+				0x80 | (charcode & 0x3f));
```

Comment:
Ignore the changes of this file in your PR since it does not related to the title of the PR.